lib/fluent/agent.rb

```diff
@@ -134,6 +134,9 @@ def add_match(type, pattern, conf)
       output.router = @event_router if output.respond_to?(:router=)
       output.configure(conf)
       @outputs << output
+      if output.respond_to?(:outputs)
```

Comment:
For safety, adding `output.is_a?(Fluent::Plugin::MultiOutput) || output.is_a?(Fluent::MultiOutput)` check is better.